.github/workflows/auto-release.yml

```
@@ -0,0 +1,217 @@
+          # Output for GitHub Actions
+          echo "release_notes<<EOF" >> $GITHUB_OUTPUT
+          cat release_notes.md >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+          
+      - name: Create Git Tag
+        run: |
+          git tag -a "v${{ steps.bump_version.outputs.new_version_name }}" -m "Release v${{ steps.bump_version.outputs.new_version_name }}"
+          git push origin "v${{ steps.bump_version.outputs.new_version_name }}"
+          
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)